### DISCUSSION TITLE
Formato filtro data creazione

### DISCUSSION THREAD

**Discussion by simsac81** [fattureincloud/api#293] [Q&A] [answered]
Salve,
sto provando ad utilizzare l'api 

**GET** /c/{company_id}/issued_documents

utilizzando un filtro sul campo created_at in modo tale da ottenere solo le fatture emesse dopo una determinata data.

Leggendo la vostra documentazione ho costruito il filtro in questo modo

**created_at%20%3E%3D%20%272023-03-07%2019%3A20%3A50%27**

Purtroppo ottengo la seguente risposta 

{
  "error": {
    "message": "Invalid query syntax.",
    "validation_result": null
  }
}

Immagino quindi che il formato della data o dell'intero filtro non sia corretto.
Ho provato altri formati data ma ottengo sempre lo stesso tipo di risposta.
Potreste indicarmi quale è il corretto formato da utilizzare per ottenere quanto desiderato?

**Comment by emajo** [comment, marked as answer]
Ciao @simsac81, il formato corretto per questo tipo di filtro è del tipo `created_at > '2023-03-07' `, facendo l'url decode del suo filtro noto che sta venendo passato anche l'orario, rimuovendo dovrebbe funzionare correttamente.

Grazie,
Emanuele.

**Comment by simsac81** [comment]
Grazie, per la risposta tempestiva.
Effettivamente senza orario funziona; avevo inserito anche l'orario in quanto nella risposta ottenuta sull'api di lista documenti quel campo contiene l'orario e quindi avevo supposto che il formato fosse lo stesso.

Grazie

**Reply by emajo** [reply]
Effettivamente il campo contiene anche l'orario, ho fatto un breve test e nel mio caso funziona anche con `created_at >= '2023-03-07 19:20:50'`, molto strano dia errore.
